### PR TITLE
[CCTRI-3239] - Implement "Affected instances" chart

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**

### CODE REVIEW

File: code/api/charts/affected_instances.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+from api.charts.factory import IChart
```
Comment (oshynk) [inline code/api/charts/affected_instances.py:1]: Please follow importing rules here and in other files

File: code/api/charts/factory.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                "tags": cls.tags,
+                "description": cls.description,
+                "short_description": cls.short_description,
+                "periods": list(cls.periods.keys()),
+                "default_period": cls.default_period
+            }
+        return [chart(cls.__call__()) for cls in IChart.__subclasses__()]
+
+
+class IChart(metaclass=ABCMeta):
```
Comment (oshynk) [inline code/api/charts/factory.py:34]: Is IChart stands for charts interface?

Comment (TarasMaliuchenko) [reply code/api/charts/factory.py:34]: Exactly

File: code/api/charts/affected_instances.py (hunk 1 above)
Comment (TarasMaliuchenko) [reply code/api/charts/affected_instances.py]: Done - 0bd7f94. Thank you.

**Review by oshynk — DISMISSED** [review]

**Review by oshynk — APPROVED** [review]

**Review by mstoro — APPROVED** [review]

**Pull request merged**